doc/dune-files.rst

```diff
@@ -2119,64 +2099,57 @@ context, or it can be the description of an opam switch, as follows:
 
 - ``(env <env>)`` sets the environment for a particular context. This is of
   higher precedence than the root ``env`` stanza in the workspace file. This
   field has the same options as the :ref:`dune-env` stanza.
 
-- ``(toolchain <findlib_toolchain>)`` sets a ``findlib`` toolchain for the context.
+- ``(toolchain <findlib_toolchain>)`` sets a ``findlib`` toolchain for the
+  context.
 
 - ``(host <host_context>)`` chooses a different context to build binaries that
   are meant to be executed on the host machine, such as preprocessors.
 
-- ``(paths (<var1> <val1>) .. (<varN> <valN>))`` allows you to set the value of any
-  ``PATH``-like variables in this context. If ``PATH`` itself is modified in
-  this way, its value will be used to resolve workspace binaries,
-  including finding the compiler and related tools. These variables will also be
-  passed as part of the environment to any program launched by Dune. For
-  each variable, the value is specified using the :ref:`ordered-set-language`.
-  Relative paths are interpreted with respect to the workspace root. See
-  :ref:`finding-root`.
-
-- ``(fdo <target_exe>)`` builds this context with feedback-direct
-  optimizations. It requires `OCamlFDO
-  <https://github.com/gretay-js/ocamlfdo>`__. ``<target_exe>`` is a
-  path-interpreted relative to the workspace root (see
-  :ref:`finding-root`). ``<target_exe>`` specifies which executable to
-  optimize. Users should define a different context for each target
-  executable built with FDO. The context name is derived
-  automatically from the default name and ``<target-exe>``, unless
-  explicitly specified using the ``(name ...)`` field.  For example, if
-  ``<target_exe>`` is *src/foo.exe* in a default context, then the
-  name of the context is *default-fdo-foo* and the filename
-  that contains execution counters is *src/fdo.exe.fdo-profile*.  This
-  feature is **experimental** and no backwards compatibility is
-  implied.
-
-- By default, Dune builds and installs dynamically-linked foreign
-  archives (usually named ``dll*.so``). It's possible to disable
-  this by setting by including
-  ``(disable_dynamically_linked_foreign_archives true)`` in the
-  workspace file, so bytecode executables will be built
-  with all foreign archives statically linked into the runtime system.
+- ``(paths (<var1> <val1>) .. (<varN> <valN>))`` allows you to set the value of
+  any ``PATH``-like variables in this context. If ``PATH`` itself is modified in
+  this way, its value will be used to resolve workspace binaries, including
+  finding the compiler and related tools. These variables will also be passed as
+  part of the environment to any program launched by Dune. For each variable,
+  the value is specified using the :ref:`ordered-set-language`. Relative paths
+  are interpreted with respect to the workspace root. See :ref:`finding-root`.
+
+- ``(fdo <target_exe>)`` builds this context with feedback-direct optimizations.
```

Comment:
Optimizations 
(Have we gotten confirmation on whether we’re using British or American spellings for Dune docs?)

Reply:
Not that I am aware of. I think we were told to be consistent, but that isn't exactly clear from the docs. Let's just use american for now.

Reply:
Interestingly, I found the British spelling of behaviour in here, so perhaps it should be British. I'll have a look at some other Dune docs to see.

Reply:
It is American spelling in the ones I saw, so I'll make a note for American spelling in Dune docs. I'll go back and correct the word to behavior, too.